__tests__/app.test.js

```diff
@@ -0,0 +1,27 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/");
+const db = require("../db/connection");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  test("GET:200 get all topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(Array.isArray(topics)).toBe(true);
+        expect(topics.length).toBe(3);
+        console.log(topics);
+        topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
```

Comment:
If your array variable `topics` has a length property, and it has access to a `forEach` method. Then it must be an array. 

So do you need to include line 16, `Array.isArray(topics)` ?